mods/force-thick-frames.wh.cpp

```diff
@@ -0,0 +1,87 @@
+// ==WindhawkMod==
+// @id              force-thick-frames
+// @name            Force thick frames
+// @description     Force windows to have thick frames like in Windows Vista/7. 
+// @version         2
+// @author          Alcatel
+// @github          https://github.com/arukateru
+// @include         *
+// @exclude         notepad++.exe
+// ==/WindhawkMod==
+
+ // ==WindhawkModReadme== 
+ /*  
+ Force windows to have thick frames like in Windows Vista/7. This theme should already have thick borders (so like, not the default Windows 10 or 11 theme)
+ ![Screenshot](https://user-images.githubusercontent.com/84914212/251890469-a04311e7-bbed-449f-9c92-79c642c36005.png) 
+ */ 
+ // ==/WindhawkModReadme== 
+ 
+ using PUNICODE_STRING = PVOID;
```

Comment:
* Remove the spaces that sneaked before each line.
* Add an empty line before the screenshot, otherwise it's all treated as a single line by markdown.
* Only images from the following domains can be embedded in the readme: https://i.imgur.com, https://raw.githubusercontent.com. See [the wiki](https://github.com/ramensoftware/windhawk/wiki/Creating-a-new-mod#readme).

After the changes, you can compile the mod and click on Preview Mod to make sure it looks correctly.